File: Easy/0058 - lengthoflastword/lengthoflastword.java
class Solution {
    public int lengthOfLastWord(String s)
    {
        int     k = 0;
        boolean flag = false;

        for (int i = s.length() - 1; i >= 0; i--)
        {
            if (s.charAt(i) == ' ' && !flag)
                continue;
            if (s.charAt(i) != ' ')
            {
                if (!flag)
                    flag = true;
                k++;
            }
            else
                break;
        }

        return k;
    }
}

File: Easy/0058 - lengthoflastword/lengthoflastword.js
var lengthOfLastWord = function(s)
{
    let     k = 0;
    let    flag = 0;

    for (let i = s.length - 1; i >= 0; i--)
    {
        if (s[i] == ' ' && !flag)
            continue;
        if (s[i] != ' ')
        {
            if (!flag)
                flag = 1;
            k++;
        }
        else
            break;
    }

    return k;    
};